sdk/python/feast/infra/registry/base_registry.py

```diff
@@ -545,22 +545,72 @@ def list_validation_references(
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def apply_project_metadata(
```

Comment:
Wouldn't it make more sense for this to also accept `ProjectMetadata` object as an input similar to other registry resources? I know the object doesn't hold much information right now, but if we plan for it to be "updatable", an update should happen through this method, right?

Reply:
I was not envisioning this as a updatable. At this point its create only. Fields we have for this are UUID and LastUpdateTimestamp. UUID should be constant. LastUpdateTimestamp should be update when any changes happen to any objects associated to the project.